bootstrap/postgres/configs.go

```diff
@@ -118,20 +124,33 @@ func (cr configRepository) RetrieveAll(key string, filter bootstrap.Filter, offs
 		c := bootstrap.Config{Owner: key}
 		if err := rows.Scan(&c.MFThing, &c.MFKey, &c.ExternalID, &c.ExternalKey, &name, &content, &c.State, &chs); err != nil {
 			cr.log.Error(fmt.Sprintf("Failed to read retrieved config due to %s", err))
```

Comment:
`Failed to retrieve configs due to`

Reply:
@manuio These logs are in sync with the corresponding ones from the Things service. Also, this error is really due to scanning, not actually retrieving, so this comment is more suitable.